operational/flexera/onboarding/flexera_onboarding.pt

```diff
@@ -0,0 +1,1918 @@
+    if (ds_service_account_clients.length == 0) {
+      // Details from official docs page
+      // https://developer.flexera.com/docs/api/iam/v1#/Service%20Account%20Client
+      details = [
+        "Each service account client belongs to a service account, and consists of a clientId and clientSecret. The clientSecret is a sensitive credential and must be stored securely. Lost or compromised clients should be deleted immediately.",
+        "Generally, only one of the service account's clients should be in use at a time. The exception is during a no-downtime credential rotation, a new client must be generated and supplied to the running application acting as a service account, before the previous client is deleted.",
+        "The service account client's clientId and clientSecret are used to authenticate with Flexera One's API. After authentication, the application's permission in the Flexera One API is determined by the roles assigned to the service account.",
+        "More details in the [Flexera API > Service Account Client Documentation](https://developer.flexera.com/docs/api/iam/v1#/Service%20Account%20Client)"
+      ]
+      check_results.push({
+        "name": "First Service Account Client",
+        "severity": "high",
+        "result": "fail",
+        "details": "No Service Account Clients found.\\n\\n"+details.join("\\n\\n")
+      });
+    } else {
+      // Check to see if there is at least one Service Account Client for at least one Service Account
+      check_results.push({
+        "name": "First Service Account Client",
+        "severity": "high",
+        "result": "pass",
+        "details": ds_service_account_clients.length+" Service Account Client(s) found."
+      });
+      //
+    }
+  }
+EOS
+end
+
+# Get Bill Connection Details
+datasource "ds_bill_connects" do
+  request do
+    auth $auth_flexera
+    host val($ds_flexera_api_hosts, 'flexera')
+    path join(["/finops-onboarding/v1/orgs/", rs_org_id, "/bill-connects"])
+  end
+  result do
+    # https://developer.flexera.com/docs/api/finops-onboarding/v1#/Bill%20Connect/Bill%20Connect%23index
+    collect jq(response, ".values[]") do
+      field "aws", jq(col_item, ".aws")
+      field "azureEa", jq(col_item, ".azureEa")
+      field "azureEaManagement", jq(col_item, ".azureEaManagement")
+      field "cbi", jq(col_item, ".cbi")
+      field "cbiAzureMca", jq(col_item, ".cbiAzureMca")
+      field "createdat", jq(col_item, ".createdAt")
+      field "gcp", jq(col_item, ".gcp")
+      field "id", jq(col_item, ".id")
+      field "kind", jq(col_item, ".kind")
+      # use jq to remove "finops:bill-connect-" from the beginning of the kind value
+      field "type", jq(col_item, '.kind | sub("finops:bill-connect-"; "")')
+      field "updatedAt", jq(col_item, ".updatedAt")
+    end
+  end
+end
+
+# Get Bill Connect Validations
+# datasource "ds_bill_connect_validations" do
```

Comment:
Is this commented out datasource still needed or does the datasource at line 210 supersede this?